keyboards/jels/boaty/config.h

```diff
@@ -0,0 +1,47 @@
+ /* Copyright 2022 Joah Nelson (Jels)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation,   either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor paramater */
+#define VENDOR_ID       0x006a     //jels
```

Comment:
While this vendor ID doesn't appear to be in use, I am very uncomfortable about using such a low ID. 

Ideally, anything past 0x4000 should be okay, and the further past, the better.

Reply:
I'd be happy to change it, but I'd also have to update my other boards (easy enough) and then also update via (could take a while) and they would be unrecognized in the down time